.github/workflows/build_steps.yml

```diff
@@ -70,10 +72,35 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3.3.0
         with:
-          submodules: recursive # Just in case a dep has its own third-party deps
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Convert to Full Clone
```

Comment:
Why do we need this now?

Reply:
Building with vcpkg requires it to be entirely cloned.

IIRC, the previous was not working on macOS, and this is an alternative which works.